integration_tests/test_tickets.py

```diff
@@ -0,0 +1,80 @@
+"""
+integration_tests.test_tickets
+----------------------------------
+
+Integration tests for the client
+"""
+import unittest
+
+from teamsupport.models import Contact, Ticket
+
+
+class TestTicket(unittest.TestCase):
+    first_name = 'Eugene'
+    last_name = 'Koval'
+    email = 'test@email.com'
+    ticket_name = 'test ticket dont touch'
+    ticket_text = 'test descr dont touch'
+
+
+class TestCreateTicket(TestTicket):
+    def setUp(self):
+        self.returned_ticket = Ticket.create(
+            self.email, self.first_name, self.last_name,
+            self.ticket_name, self.ticket_text)
+        self.ticket = Ticket(self.returned_ticket.id)
+        self.contact = Contact.get(self.email)
+
+    def test_ticket_is_created(self):
+        self.assertEqual(self.ticket.Name, self.ticket_name)
+
+    def test_ticket_description_is_set(self):
+        description = self.ticket.get_description()
+        self.assertEqual(description, self.ticket_text)
+
+    def test_new_contact_is_created(self):
+        self.assertEqual(self.contact.FirstName, self.first_name)
+        self.assertEqual(self.contact.LastName, self.last_name)
+        self.assertEqual(self.contact.Email, self.email)
```

Comment:
> Otherwise eugene will never be able to file tickets :P

I don't have TS account at all. If Support wants to file a ticket associated with me, they will be able to create a contact with my email.